package.json

```diff
@@ -31,14 +31,13 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-datetime": "^1.5.0",
-    "coffeescript": "^2.5.0",
-    "husky": "^4.2.0",
-    "lint-staged": "^10.0.2",
+    "coffeescript": "^2.5.1",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.0.8",
     "mocha": "^6.2.2",
-    "prettier": "^1.19.1",
     "require-npm4-to-publish": "^1.0.0",
-    "resin-lint": "^2.0.1",
-    "typescript": "^3.7.5"
+    "resin-lint": "^3.3.1",
```

Comment:
resin-lint has been renamed, so if you could switch to `@balena/lint @ 4.0.0` it would be great.

Reply:
It doesn't install in the .bin properly atm so that needs fixing before it can be moved to